new-packages/ts-project/__tests__/actions.test.ts

```diff
@@ -0,0 +1,691 @@
+import { expect, test } from 'vitest';
+import { createTestProject, replaceUniqueIds, testdir, ts } from './utils';
+
+test('should extract a string entry action (direct)', async () => {
+  const tmpPath = await testdir({
+    'tsconfig.json': JSON.stringify({}),
+    'index.ts': ts`
+      import { createMachine } from "xstate";
+
+      createMachine({
+        entry: "callDavid",
+      });
+    `,
+  });
+
+  const project = await createTestProject(tmpPath);
+  expect(replaceUniqueIds(project.extractMachines('index.ts')))
+    .toMatchInlineSnapshot(`
+      [
+        [
+          {
+            "blocks": {
+              "block-0": {
+                "blockType": "action",
+                "parentId": "state-0",
+                "properties": {
+                  "params": {},
+                  "type": "callDavid",
+                },
+                "sourceId": "callDavid",
+                "uniqueId": "block-0",
+              },
+            },
+            "edges": {},
+            "implementations": {
+              "actions": {
+                "undefined": {
```

Comment:
The object key here should be "callDavid".

Reply:
Ah, good catch - that was a bug in the test utils. I just pushed out a fix for this.